projects/plugins/crm/src/js/components/automations/view.tsx

```diff
@@ -0,0 +1,23 @@
+import * as WPElement from '@wordpress/element';
+import { AutomationsAdmin } from '.';
+
+/**
+ * Render function
+ */
+const render = () => {
+	const container = document.getElementById( 'jetpack-crm-automations-root' );
+
+	if ( null === container ) {
+		return;
+	}
+
+	const component = <AutomationsAdmin />;
```

Comment:
AFAIK then `ThemeProvider` should make sure that we have support/access to all Jetpack colors (e.g.: `--jp-black-80`)

```suggestion
	// @todo: Remove fallback when we drop support for WP 6.1
	const component = (
		<ThemeProvider>
			<AutomationsAdmin />
		</ThemeProvider>
	);
```

Reply:
I'm not exactly sure what `ThemeProvider` is giving us as I still need to have `@import '../../../../sass/_color_studio.scss';` in the scss file in order for it to compile, but I added it anyway.

Reply:
`<ThemeProvider>` makes sure that the `<AutomationsAdmin>` component can use the CSS variables Jetpack exposes:

![Screenshot 2023-07-18 at 15 41 28](https://github.com/Automattic/jetpack/assets/3846700/06e735d8-582d-430e-aa3b-d5a368aff7ee)